Includes optional what3words address of the workshop venue

pgmccann opened the pull request:
This change adds an optional [what3words](https://what3words.com) address field to the index page, providing organisers with another, user-friendly way of specifying the venue location.

pgmccann committed:
Includes optional what3words address of the workshop venue

tobyhodges reviewed (approved):
I like this suggestion, thank you @pgmccann. Do you think people will understand why/when using W3W might be beneficial, or should we provide more explanation of the circumstance sin which the additional, more precise location info could be helpful?

pgmccann committed:
Corrects what3words URL in variable description

pgmccann commented:
I think that's a good question.

The comment next to the variable assignment in the header currently contains about as much information as the others around it - I'm not sure a longer description is appropriate for an optional value. I also fear that an explanation in the text of the page might get in the way of the location information itself.

Would a simple

> (find out more about [what3words](https://what3words.com/about))

after the location information in the page suffice?

tobyhodges commented:
Please could you add a description of the new variable on [the Customization page](https://carpentries.github.io/workshop-template/customization/index.html)? I don't think that page is widely known, but it is the appropriate place for a more detailed description of the various configuration options.

[Edit: source file for that page in the template: https://github.com/carpentries/workshop-template/blob/gh-pages/_extras/customization.md]

pgmccann committed:
Adds description of what3words and why it might be useful.

Moves the what3words variable to sit alongside the other optional
variables, for consistency with the documentation.

tobyhodges reviewed (approved):
Fantastic, thank you @pgmccann. (Thanks also for your patience while you waited for me to review your pull requests 🥲)
